### DISCUSSION TITLE
Theme color based favicon

### DISCUSSION THREAD

**Discussion by ilkr** [webmin/authentic-theme#1615] [Ideas]
Is it possible to change the favicon color with to the authentic theme color theme.

**Comment by iliaross** [comment]
Hello,

This is interesting suggestion, and also reasonable .. I am not sure if that can be done easily .. I will have to check ..

**Reply by ilkr** [reply]
Hello Ilia, if I remember correctly, I made the suggestion of color palette 5-6 years ago, I'm sure you could do it. Kind regards
